src/app/components/challenge/challengesettings/challengesettings.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit } from '@angular/core';
+import { ChallengeService } from '../../../services/challenge.service';
+import { EndpointsService } from '../../../services/endpoints.service';
+import { ApiService } from '../../../services/api.service';
+import { GlobalService } from '../../../services/global.service';
+
+@Component({
+  selector: 'app-challengesettings',
+  templateUrl: './challengesettings.component.html',
+  styleUrls: ['./challengesettings.component.scss']
+})
+export class ChallengesettingsComponent implements OnInit {
+
+  /**
+   * Challenge object
+   */
+  challenge: any;
+
+  /**
+   * Participants banned emails ids
+   */
+  bannedEmailIds: string[];
+
+  /**
+   * Input to edit the banned participants emails
+   */
+  isBannedEmailInputVisible: boolean;
+
+  constructor(private challengeService: ChallengeService, private globalService: GlobalService,
+              private apiService: ApiService, private endpointsService: EndpointsService) { }
+
+  ngOnInit() {
+    this.challengeService.currentChallenge.subscribe(
+      challenge => {
+        this.challenge = challenge;
+        this.updateView();
+    });
+  }
+
+  updateView() {
+    this.bannedEmailIds = this.challenge.banned_email_ids;
+  }
+
+  getBannedEmailsListObject(bannedEmailInputValue) {
+    const bannedEmailIds = [];
+    // First remove all the whitespaces and then split
+    const splittedEmails = bannedEmailInputValue.split(',').map(item => item.trim());
+    for (let i = 0; i < splittedEmails.length; i++) {
+      bannedEmailIds.push(splittedEmails[i]);
```

Comment:
Should we also apply an email `regex` check for each email?

Reply:
@lunayach done